src/main/java/org/jboss/xavier/analytics/pojo/output/InitialSavingsEstimationReportModel.java

```diff
@@ -11,21 +11,31 @@
 import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.Index;
 import javax.persistence.JoinColumn;
 import javax.persistence.MapsId;
 import javax.persistence.OneToOne;
+import javax.persistence.Table;
 import java.util.Date;
 
 /**
  * This class was automatically generated by the data modeler tool.
  */
 
 @Entity
+@Table(
+        indexes = {
+                @Index(name = "InitialSavingsEstimationReportModel_" +
+                        InitialSavingsEstimationReportModel.ANALYSIS_ID + "_index",
+                        columnList = InitialSavingsEstimationReportModel.ANALYSIS_ID, unique = false)
```

Comment:
`unique` value is by default `false` so no need to write it again.

Reply:
TBH i think it's better to set the value we want without relying on default value in this case in order to avoid any issue if and when the default value will change in the future.